pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -32,20 +32,29 @@
 class Ametek7270(Instrument):
     """This is the class for the Ametek DSP 7270 lockin amplifier"""
 
+    SENSITIVITIES = [

```

Comment:
This looks good! Having them as class variables makes them more accessible to other uses.